### PR TITLE
[@types/chrome] Update chrome.tabs API to add missing parameter and function for tabgroup use

### PR DESCRIPTION
**Pull request by MrSweeter**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://developer.chrome.com/docs/extensions/reference/tabs/>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/chrome/index.d.ts [hunk 1]
```diff
@@ -8452,6 +8457,13 @@ declare namespace chrome.tabs {
      * @param callback Optional.
      */
     export function group(options: GroupOptions, callback?: (groupId: number) => void): void
+    /**
+     * Removes one or more tabs from their respective groups. If any groups become empty, they are deleted
+     * @since Chrome 88
+     * @param tabIds The tabs to ungroup.
+     * @param callback Optional. Called after the operation is completed.
+     */
+    export function ungroup(tabIds: number | number[], callback?: () => void): void
```
Comment (matcarlson) [inline types/chrome/index.d.ts:8466]: Can you add an override for this function returning a Promise, similar to [line 8341](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/chrome/index.d.ts#L8341)

File: types/chrome/index.d.ts [hunk 2]
```diff
@@ -8452,6 +8457,13 @@ declare namespace chrome.tabs {
      * @param callback Optional.
      */
     export function group(options: GroupOptions, callback?: (groupId: number) => void): void
```
Comment (matcarlson) [inline types/chrome/index.d.ts:8459]: Since this is related, could you also add the Promise override here as well?

**Review by matcarlson — CHANGES_REQUESTED** [review]

**Review by matcarlson — APPROVED** [review]
Thanks for the changes!

**Review by userTim — APPROVED** [review]

**Comment by userTim** [comment]
Ready to merge

**Pull request merged**